DCScribe/Core/Scribe.cs

```diff
@@ -190,12 +191,18 @@ private async Task ProcessMarkPanelUpdates(CancellationToken scribeToken)
                     _logger.LogInformation("{server} Writing {count} markpanel(s) to database ", GameServer.ShortName, markPanels.Count);
                     await _databaseClient.TruncateMarkPanelsAsync();
                     await _databaseClient.WriteMarkPanelsAsync(markPanels, scribeToken);
-                    await Task.Delay(TimeSpan.FromSeconds(900), scribeToken);
+                    await Task.Delay(TimeSpan.FromSeconds(GetTaskTimer("ProcessMarkPanelUpdates")), scribeToken);
                 } catch (Exception)
                 {
                     // No-op. Exceptions have already been logged in the task
                 }
             }
         }
+
+        private int GetTaskTimer(string taskName)
```

Comment:
This gets a lot simpler if we have named fields instead of a list. We can probably get rid of it and just reference `GameServer.RecordUnitTask.Timer` and the like directly.

As a sidenote: I don't think it matters anymore since we will get rid of this but I recommend using constants instead of passing strings for values known ahead of time like config variables.

Reply:
This is why I'm the contributor and you're the maintainer. :-)
I appreciate the sidenote, will take it on-board.